modules/SofaMiscFem/src/SofaMiscFem/TriangularFEMForceField.inl

```diff
@@ -200,41 +211,41 @@ void TriangularFEMForceField<DataTypes>::initLarge(int i, Index&a, Index&b, Inde
     }
     else
     {
+        VecCoord initialPoints = (this->mstate->read(core::ConstVecCoordId::restPosition())->getValue());
+        tinfo->rotation = tinfo->initialTransformation;
+        if (a >= (initialPoints).size() || b >= (initialPoints).size() || c >= (initialPoints).size())
+        {
+            msg_error() << "Try to access an element which indices bigger than the size of the vector: a=" << a <<
+                " b=" << b << " and c=" << c << " and size=" << (initialPoints).size() << msgendl;
+            triangleInfo.endEdit();
```

Comment:
The use of endEdit() in a condition should probably be handled more elegently with an accessor.